stitches/tests/test_internal_data.py

```diff
@@ -3,65 +3,57 @@
 import os
 import unittest
 
+from stitches.fx_util import nrow
 from stitches.fx_pangeo import fetch_pangeo_table
 
-class MyInternalData(unittest.TestCase):
+class TestInternalData(unittest.TestCase):
 
+    # This test checks the internal package data and throws an error if the
+    # internal package data needs to be updated/rerun.
+    PANGEO_TABLE = pd.read_csv(pkg_resources.resource_filename('stitches', 'data/pangeo_comparison_table.csv'))
+    MATCHING_ARCHIVE = pd.read_csv(pkg_resources.resource_filename('stitches', 'data/matching_archive.csv'))
 
-    def test_pkg_data(self):
+    # Get the most up todate pangeo catalog
+    CURRENT_DATA = fetch_pangeo_table()
 
-        # Make sure the matching data is consistent with internal tas data.
-        #
-        # Read in the archive that is used in the matching process, we are going to
-        # compare the model / experiment / ensembles / variables that exist here with
-        # must also exist in the raw tas data files.
-        path = pkg_resources.resource_filename('stitches', 'data/matching_archive.csv')
-        df = pd.read_csv(path)[["experiment", "model", "ensemble"]]
-        matching_archive = df.drop_duplicates(keep='last')
+    def test_pangeo_catalog1(self):
+        internal_fresh = self.PANGEO_TABLE.merge(self.CURRENT_DATA, how='left', indicator=True)
+        new_data = internal_fresh[internal_fresh["_merge"] == 'left_only']
+        self.assertEqual(nrow(new_data), 0, "New data available on Pangeo")
+        new_tas_data = new_data[new_data["variable_id"] == "tas"]
+        self.assertEqual(nrow(new_tas_data), 0, "New tas available on Pangeo")
```

Comment:
@abigailsnyder should we trigger the new data is available error whenever any file is added to pangeo or should it just be for if new tas files are added?

Reply:
I guess it goes back to the question we've been thinking about, whether we want to have package data in zenodo and ship that with the release or just have the default behavior be to run building the package data first.

Reply:
^^ the ship or build data will forever be a big question... hmm we could change the name of the python file so it is not triggered as a unit test but we still have it floating around until we figure out how we want to deal with the data stuff